app/javascript/components/user-menu/user-menu.jsx

```diff
@@ -0,0 +1,61 @@
+import {faCog, faSignInAlt} from '@fortawesome/free-solid-svg-icons';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import React, {useState} from 'react';
+import {useNavigate} from 'react-router-dom';
+import {Avatar} from '../avatar/avatar';
+import style from './style.module.css';
+
+export const UserMenu = () => {
+  const history = useNavigate();
+  const [isOpenMenu, setIsOpenMenu] = useState(false);
+
+  return (
+    <>
```

Comment:
```suggestion
    <div
      onMouseLeave={() => {
        setIsOpenMenu(false);
      }}
    >
```

и изменить `margin: 32px` на `padding 32px ` в  .userMenu /user-menu/style.module.css
тогда меню будет закрываться автоматически на mouseLeave